### ISSUE TITLE
ValueError: bad marshal data (unknown type code)

### ISSUE THREAD

**Issue by maximilianchang** [codelucas/newspaper#527]
When I run: "sudo python3 setup.py install", I get the following error: 
<img width="786" alt="screen shot 2018-03-08 at 10 28 51 pm" src="https://user-images.githubusercontent.com/10934170/37193774-1bf546b2-2320-11e8-9df9-411cd23a2e0e.png">
From reading online, I tried removing all the *.pyc files, to no result. I noted the old issue that discussed this, but that seemed resolved. 

The strange thing is that it used to work, but more recently, it has failed to work on my VM (although it still works on my MAC). I'm running Debian 9 on my VM.

**Comment by simonkeng** [comment]
I am getting this as well. I'm on Raspbian. I also tried removing complied python files (per stackoverflow result) but this didn't solve it for me either. `pip install` newspaper works perfectly on my macOS machine, but I'm having trouble on linux.

**Comment by jessefogarty** [comment]
Have you tried pip install newspaper3k- Jesse
-- 
Securely sent with Tutanota.

If you forgot your conversation password. Please text me using Signal @ (647)571-7808.



Feb. 20, 2019, 11:28 p.m. by notifications@github.com:

**Comment by simonkeng** [comment]
Yes, like I said in my update, the regular `pip install newspaper3k` worked great and avoided this issue. Might consider closing this issue for now.

**Comment by toshiro92** [comment]
For those who still facing this issue when manually install newspaper in `Python3`, there is the command you have to do to solve this problem:

`pip3 install git+https://github.com/codelucas/newspaper.git`

Source: https://stackoverflow.com/questions/30650098/error-in-installing-newspaper3k-package-via-pip
